### PR TITLE
Upgrade console to use operator v4.0.0

### PR DESCRIPTION
**Pull request by dvaldivia**
Updates all the usage of the operator to the new v4.0.0 structure which uses Tenant V2 and Pools

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,7 +30,9 @@ require (
...
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.8
-	k8s.io/client-go v0.18.6
+	k8s.io/api v0.20.2
+	k8s.io/apimachinery v0.20.2
+	k8s.io/client-go v0.20.2
 )
+
+replace github.com/minio/operator v0.0.0-20210127071132-accfa4851396 => github.com/dvaldivia/operator v0.0.0-20210128022015-0ce833ee3034
```
Comment (cesnietor) [inline go.mod:38]: no no

**Review by cesnietor — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (dvaldivia) [reply go.mod:38]: hahaha, it's a "place holder"

Comment (dvaldivia) [reply go.mod:38]: tenor-6

Comment (harshavardhana) [reply go.mod:38]: placeholder can be removed now @dvaldivia

Comment (dvaldivia) [reply go.mod]: @harshavardhana removed

**Review by cesnietor — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**